sdk/spring/CHANGELOG.md

```diff
@@ -1,49 +1,151 @@
 # Release History
 
 ## 4.0.0-beta.4 (Unreleased)
-- Support Spring Boot version: 2.5.5+ and 2.6.0 - 2.6.3.
-- Support Spring Cloud version: 2020.0.3+ and 2021.0.0.
+- Support Spring Boot version: 2.5.5+ and 2.6.0+.
+- Support Spring Cloud version: 2020.0.3+ and 2021.0.0+.
 
 ### Dependency Updates
 - Upgrade dependency according to spring-boot-dependencies:2.6.3 and spring-cloud-dependencies:2021.0.0.
 
+### Features Added 
+- Add `Automatic-Module-Name` for all Spring Cloud Azure modules and change the root pacakge names to match the module names [#27350](https://github.com/Azure/azure-sdk-for-java/issues/27350), [#27420](https://github.com/Azure/azure-sdk-for-java/pull/27420).
+
+### Spring Cloud Azure Dependencies (BOM)
+#### Dependency Updates
+- Use `azure-sdk-bom:1.2.0` instead of including each sdk dependencies [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
+- Use `azure-cosmos:4.28.0` instead of the version `4.27.0` in `azure-sdk-bom` [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
+- Use `azure-resourcemanager:2.13.0`.
+- Use `azure-spring-data-cosmos:3.19.0`.
```

Comment:
```
 - Import bom: `azure-sdk-bom:1.2.0`. [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
 - Delete `aaa, bbb, ...` because these versions already specified in `azure-sdk-bom:1.2.0`. [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
 - Add new dependency `ccc`. [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
 - Upgrade dependency`ddd` to `version-ddd` . [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
```